src/components/input-group/input-group.stories.js

```diff
@@ -0,0 +1,80 @@
+import { html } from "lit-html";
+import "../input-group";
+import "../button";
+import "../input-field";
+import "../input-addon";
+import "../icon";
+import "../text";
+import styles from "./styles.scss";
+import buttonStyles from "../button/styles.scss";
+import inputFieldStyles from "../input-field/styles.scss";
+import inputAddonStyles from "../input-addon/styles.scss";
+import iconStyles from "../icon/styles.scss";
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+
+export default {
+  title: "native-elements/vl-input-group",
+  decorators: [
+    (story) =>
+      html`${stylesheet(
+        `${styles}${inputFieldStyles}${iconStyles}${inputAddonStyles}${buttonStyles}`
+      )}${story()}`,
+  ],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
```

Comment:
In de docs staat nu dat de input-group geen stylesheet nodig heeft, dit lijkt wel zo te zijn.

Reply:
Fixed.